recipe/meta.yaml

```diff
@@ -21,20 +21,17 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<37]
+  skip: True  # [py<38]
   skip: True  # [linux and (s390x or ppc64le)]
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
-    - setuptools
-    - wheel
   run:
-    - python
+    - python >=3.6
     - pip    >=18.0
     - setuptools
-    - wheel
     - anyqt >=0.1.0
     - pyqt
     - docutils
```

Comment:
Modify `cachecontrol >=0.12.6` as `cachecontrol-with-filecache >=0.12.6`

Reply:
As we do not build for py<38 you can remove `importlib_metadata` https://github.com/biolab/orange-canvas-core/blob/4f4a7c6f0f503ed3a11cc361c40889c1293dda95/setup.py#L34

Reply:
Looks like `lockfile` was added 4 years ago https://github.com/conda-forge/orange-canvas-core-feedstock/commit/0e25c359e926a8c4553c317af597b865322eea7d and now it's not mentioned in the setup.py. Could you verify that it's really needed for runtime or it's a redundant one?

Reply:
> Looks like `lockfile` was added 4 years ago [conda-forge/orange-canvas-core-feedstock@0e25c35](https://github.com/conda-forge/orange-canvas-core-feedstock/commit/0e25c359e926a8c4553c317af597b865322eea7d) and now it's not mentioned in the setup.py. Could you verify that it's really needed for runtime or it's a redundant one?

how do I do that?